tidy3d/plugins/dispersion/fit_web.py

```diff
@@ -122,71 +192,74 @@ def fit(  # pylint:disable=arguments-differ, too-many-arguments, too-many-locals
             Number of optimizations to run with random initial guess.
         tolerance_rms : NonNegativeFloat, optional
             RMS error below which the fit is successful and the result is returned.
-        bound_amp : PositiveFloat, optional
-            Bound on the amplitude of poles, namely on Re[c], Im[c], Re[a]
-        bound_f : PositiveFloat, optional
-            Bound on the frequency of poles, namely on Im[a]. Unit: Hz
-        constraint : Literal["hard", "soft"], optional
-            'hard' or 'soft'
-        nlopt_maxeval : PositiveInt, optional
-            Maxeval in each optimization
-
-        Returned
-        ------------------
+        advanced_param : :class:`AdvancedFitterParam`, optional
+            Other advanced parameters.
+
+        Returns
+        -------
         Tuple[``PoleResidue``, float]
             Best results of multiple fits: (dispersive medium, RMS error).
         """
 
-        from ...web.auth import (
-            get_credentials,
-        )  # pylint:disable=import-outside-toplevel, unused-import
-        from ...web.httputils import get_headers  # pylint:disable=import-outside-toplevel
+        # get url
+        url_server = self._set_url("default")
+        headers = self._setup_server(url_server)
 
-        _url = self._set_url("default")
+        # set up bound_f, bound_amp
+        if advanced_param.bound_f is None:
+            advanced_param.bound_f = self.frequency_range[1] * 10
```

Comment:
instead of hardcoding this `10`, let's make it either a constant `BOUND_MAX_FACTOR = 10` defined somewhere or an argument to this function if it makes more sense.